### ISSUE TITLE
Label images on WORKFORCE page

### ISSUE THREAD

**Issue by VarunKumar-Xaviers** [xxchanjotxx/malhar-live#14]
The images that are present on the screen towards the left of the main text are not described. The screen reader is unable to read any content inside the image.
